terraform/main.tf

```
@@ -304,6 +304,12 @@ variable "minecraft__idea_reaction_redmine_api_key" {
   sensitive   = true
 }
 
+variable "minecraft__babyrite_discord_token" {
```

```suggestion
variable "discord_bot__babyrite_discord_token" {
```
`minecraft__`というプレフィックスは適切ではないように思われます。このように変更してみてはいかがでしょうか。

---

TODO: idea-reaction 側も修正する必要があります

---

Fixed https://github.com/GiganticMinecraft/seichi_infra/pull/2142/commits/5add88bb2d56adf599ea1985598e63445b2a0fcf